source/features/tag-changelog-link.css

```diff
@@ -0,0 +1,3 @@
+.release div.release-header > ul > li {
+	flex-grow: 0 !important;
```

Comment:
I understand that this is to avoid this issue:

<img width="664" alt="" src="https://user-images.githubusercontent.com/1402241/57436361-183d0080-7272-11e9-98bc-1d6b7f658478.png">


But this isn't right:

<img width="689" alt="Screenshot 2019-05-09 at 15 50 10" src="https://user-images.githubusercontent.com/1402241/57436439-3f93cd80-7272-11e9-8ea5-5d65802a5643.png">


The solution might be a bit ugly, but I think you should:

1. add the link after that `flex-auto`
2. remove the `flex-auto` class
3. add the `flex-auto` class to the `Changelog` link

Maybe you can do this like:

```js
if (position.is('.flex-auto')) {
  position.classList.remove('flex-auto');
  button.classList.add('flex-auto');
  position.after(button)
}
```